watertap/examples/case_studies/municipal_treatment/tests/test_municipal_treatment.py

```diff
@@ -0,0 +1,128 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import pytest
+from io import StringIO
+from pyomo.environ import value
+from watertap.examples.case_studies.municipal_treatment.municipal_treatment import main
+from idaes.core.util import get_solver
+
+# -----------------------------------------------------------------------------
+@pytest.mark.component
+def test_municipal_treatment():
+    m = main()
+
+    report_io = StringIO()
+    m.fs.feed.report(ostream=report_io)
+    output = \
+        """
+====================================================================================
+Unit : fs.feed                                                             Time: 0.0
+------------------------------------------------------------------------------------
+    Stream Table
+                             Outlet 
+    Mass Concentration H2O    999.36
+    Mass Concentration tds   0.63000
+    Mass Concentration toc 0.0040000
+    Mass Concentration tss 0.0065250
+    Volumetric Flowrate      0.92240
+====================================================================================
+"""
+    assert output == report_io.getvalue()
+
+    report_io = StringIO()
```

Comment:
Not really important following my previous comment, but you do not need to create a new `StringIO` for each step - you can just use the previous one and the new output should be appended. The only change you would need is to make the final check an `in` rather than `==`.

Reply:
I got rid of the string comparisons